src/components/CollectionImageSwiper/index.tsx

```diff
@@ -49,7 +49,7 @@ const slides: SlidesProps[] = [
 
 const CollectionImageSwiper = () => {
   return (
-    <div className={`${styles.swiper} main-page-swiper`}>
+    <div className={`${styles.swiper} collection-swiper`}>
```

Comment:
use clsx wrapper
`<div className={getValidClassNames(styles.swiper, 'collection-swiper')}>`

Reply:
https://janithrs.medium.com/conditional-class-rendering-in-react-using-clsx-b3bed12014cb

Reply:
Thank you for link. However why should we use clsx?

Reply:
Using clsx is a common practice to enhance code readability and in some cases, it improves performance.